### PR TITLE
darkhttpd: update to 1.14.

### PR DESCRIPTION
**Pull request by kruceter**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/darkhttpd/files/darkhttpd/run [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 [ -r conf ] && . ./conf
+[ -d "/var/log/darkhttpd" ] || mkdir /var/log/darkhttpd
```
Comment (classabbyamp) [inline srcpkgs/darkhttpd/files/darkhttpd/run:3]: why is this needed if it's already in make_dirs?

Comment (kruceter) [reply srcpkgs/darkhttpd/files/darkhttpd/run:3]: Good grief you mentioned make_dirs because I have not noticed it there.

**Comment by icp1994** [comment]
/archive/refs/tags
